tests/integration/test_keeper_four_word_command/test.py

```diff
@@ -641,3 +642,15 @@ def test_cmd_lgif(started_cluster):
         assert int(result["last_snapshot_idx"]) >= 1
     finally:
         destroy_zk_client(zk)
+
+
+def test_cmd_rqld(started_cluster):
+    for node in [node1, node2, node3]:
+        data = keeper_utils.send_4lw_cmd(cluster, node, cmd="rqld")
+        assert data == "Sent leadership request to leader."
+        if not keeper_utils.is_leader(cluster, node):
+            # pull wait to become leader
+            start = datetime.now()
+            while datatime.now() - start <= 2:
+                time.sleep(0.1)
+            assert keeper_utils.is_leader(cluster, node)
```

Comment:
this can be a single while loop
similar to `keeper_utils.wait_until_connected`. You can even assert False after a certain amount of retries but be careful you don't restrict it too much